### DISCUSSION TITLE
I have a lot of projects, but there are too few projects that can be pinned on the homepage, and now I can only pin 6

### DISCUSSION THREAD

**Discussion by 1107012776** [community/community#29653] [Other Feature Feedback, Questions, & Ideas]
I have a lot of projects, but there are too few projects that can be pinned on the homepage, and now I can only pin 6
我的项目很多，但是能钉在首页的项目太少了，现在只能钉6个

**Comment by mazkdevf** [comment]
Ye 

![image](https://user-images.githubusercontent.com/79049205/184849677-a311f417-e9e4-42dd-b3f8-a0a661fddeea.png)

**Reply by mazkdevf** [reply]
So at the moment you cannot ping anymore than 6

**Comment by Hellzbellz123** [comment]
https://github.com/anuraghazra/github-readme-stats#github-extra-pins i use this to pin repos, =unlimited afaik, i dont have enough pinned too make use of 6 but it exists, it needs to be added to a profile readme tho

**Reply by mazkdevf** [reply]
Yeah, hmm. Maybe GitHub will increase that in the future. (to add more pins)

**Comment by jerone** [comment]
On your personal profile readme you can add a lot of information: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

**Comment by NikhilMath** [comment]
Make a personal readme

**Comment by andresWeitzel** [comment]
Hello, as @mazkdevf has mentioned, it is only possible to pin 6 pins, but there is a way in which you can modify your profile so that you have more than 6, you should create a file called [README](https://www.nobledesktop.com/learn/git/create-a-readme-file)  and there you can add more pinned using [OPEN SOURCE METRICS](https://github.com/lowlighter/metrics/blob/master/source/plugins/repositories/README.md) . There are more than 100 of them.

**Pinned View**

![metrics plugin repositories pinned](https://user-images.githubusercontent.com/58787260/186039496-05061c26-17fa-4e9b-9583-f38c24d80283.svg)

**Comment by mmqeth** [comment]
You can create a repository with your own ID, write readme.md, and it will be your personal page.Then, you can insert 
[! [Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=0xmimiQ&repo=eb_supporter&bg_color=0d1116&title_color=ce09ec &text_color=a4aacb&icon_color=007ec6)](https://github.com/0xmimiQ/eb_supporter) 
Remember to change the repository name and ID to you want.
